docs/docusaurus.config.ts

```diff
@@ -82,6 +85,103 @@ const config: Config = {
       },
     ],
     */
+    async function pluginLlmsTxt(context) {
```

Comment:
suggestion: if this is from prisma, might be worth putting a link to the original here.

Reply:
done